functional-tests/src/test/groovy/FrontendMinespace/pages/KeycloakLoginPage.groovy

```diff
@@ -0,0 +1,12 @@
+package publicFrontend.pages
```

Comment:
Can you refactor  the package to follow similar patterns with the directory . names? i.e. `minespace` or something instead of `publicFrontend`

Reply:
refactored